test/lev-top-nav/Index.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import 'jest-styled-components';
+import LevTopNav from "../../src/lev-top-nav/index";
+import renderer from 'react-test-renderer';
+
+describe('component', () => {
+  let tree, root;
+  beforeEach(() => {
+    tree = renderer.create(<LevTopNav />);
+    root = tree.root;
+  });
+
+  it('renders correctly', () => expect(tree.toJSON()).toMatchSnapshot());
+
+  it('contains 3 links with page-link class', () => {
+    let links = root.findAllByProps({className: "page-link"});
+    expect(links.length).toBe(3);
+  })
+
+  it('contains a report link', () => {
+    let link = root.findByProps({className: "page-link", href: '/'});
```

Comment:
Urgh, yeh that's sort of hideous, but overall you've got tests for what you've added so :+1: :heavy_check_mark: VG